### DISCUSSION TITLE
How can i create a shadow effect in a stacked column chart?

### DISCUSSION THREAD

**Discussion by mustfaunlu** [patrykandpatrick/vico#627] [Questions] [answered]
How can I create a shadow effect like the one shown in the image below when I click on any column in a stacked column chart?

![Ekran görüntüsü 2024-03-24 152306](https://github.com/patrykandpatrick/vico/assets/38860392/ea90a41d-5166-4a53-ad3d-09ae4db117b8)

**Comment by mustfaunlu** [comment]
@patrickmichalik Do you have any idea?

**Reply by Gowsky** [reply, marked as answer]
Hello, @mustfaunlu. Please don't ping the maintainers. Currently, you can add shadows to individual `LineComponent`s but not to stacks as a whole. Therefore, you'd have to create a `ColumnCartesianLayer` subclass or stack two `ColumnCartesianLayer`s (the rear one would have a single series that sums the front one's series). To implement column selection and highlighting, you can use `CartesianMarkerVisibilityListener` and `ColumnCartesianLayer.ColumnProvider`.
